src/unittestScripts.ts

```diff
@@ -27,23 +27,22 @@ def get_tests(suite):
         return [suite]
 
 
-def discover_tests():
+def discover_with_loader():
```

Comment:
How about renaming it to something like `discover_suits`? IMO it better describes this method.

Reply:
yeah that's fair enough - i will note though that it's `suites` and not `suits` (i didn't take the liberty of changing the misspelling on the typescript)